src/runtime_src/core/tools/xbtracer/src/lib/capture.h

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2024 Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include "experimental/xrt_elf.h"
+#include "experimental/xrt_hw_context.h"
+#include "experimental/xrt_ip.h"
+#include "experimental/xrt_module.h"
+#include "experimental/xrt_system.h"
+#include "experimental/xrt_xclbin.h"
+#include "xrt/xrt_bo.h"
+#include "xrt/xrt_device.h"
+#include "xrt/xrt_graph.h"
+#include "xrt/xrt_kernel.h"
```

Comment:
Include-what-you-use.   Never include more headers than what is needed to compile the file (in this case capture.h) into an object file.   The source (.cpp) files include what these headers if they use them.